typescript/src/codeActions/custom/React/conditionalRendering.ts

```diff
@@ -0,0 +1,37 @@
+import { CodeAction } from '../../getCodeActions'
+
+/*
+    Before: <div></div>
+    After: { && (<div></div>)}
+*/
+export default {
+    id: 'conditionalRendering',
+    name: 'Wrap into Condition',
+    kind: 'refactor.rewrite.conditionalRendering',
```

Comment:
Vscode has surround kind for code actions so maybe use something like `refactor.surround.with.`